email.go

```diff
@@ -115,6 +115,11 @@ func (c emailConfig) sendEmail(subject string, body string, address string) erro
 	}
 	message := fmt.Sprintf("From: %s\nTo: %s\nSubject: %s\n\n%s",
 		c.sender, address, subject, body)
+	if c.submissionHostname == "" {
+		fmt.Println("Warning: email host not configured, not sending email")
+		fmt.Println(message)
```

Comment:
Ah, I think both this and the `fmt` above should be `log` prints? wdyt?

Reply:
Good catch! I think log sounds good - more consistent and seems more flexible.